src/UniswapV2Callee.sol

```diff
@@ -1,5 +1,6 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
According to the other file's convention this file should be named UniswapV2CalleeDai.sol

Reply:
I'll do that in a different PR as renaming a file makes it difficult to review changes